docs/getting-started/index.md

```diff
@@ -1,603 +1,240 @@
 
-To take full control over the appearance of the Kendo UI for Vue components, you can create your own styles by using [ThemeBuilder](slug:themebuilder).
+## Configure the Vue Data Grid
 
-ThemeBuilder is a web application that enables you to create new themes and customize existing ones. Every change that you make is visualized almost instantly. Once you are done styling the Vue components, you can export a zip file with the styles for your theme and use them in your Vue app.
+Now that you have a running Grid, you are ready to use some of its basic features like sorting and paging:
 
-### UI Kits for Figma
+1. In the Grid declaration, add [paging](slug:paging_grid), [sorting](slug:sorting_grid), and a height style that activates [scrolling](slug:scrollmmodes_grid).
 
-Kendo UI for Vue comes with [four UI Kits for Figma](slug:ui_kits_figma): Material, Bootstrap, Fluent, and Kendo UI Default. They provide the designers of your application with a building block that matches the UI components available in the Kendo UI for Vue suite. Having matching building blocks guarantees the smooth implementation of the design.
+    ```html
+      <template>
+          <h1>Hello Kendo UI for Vue!</h1>
+          <grid 
+            :data-items="products"
+            :columns="columns"
+            :pageable="pageable"
+            :sortable="sortable"
+            :style="{ height: '400px' }"
+          ></grid>
+      </template>
+    ```
+1. Implement the paging and sorting functionality in the `data` option:
+
+  * Set the [page size (`take`)](slug:api_grid_gridprops#toc-take) to 10.
+  * Set the initial [`skip`](slug:api_grid_gridprops#toc-skip) for the paging.
+  * Set the initial [sorting](slug:api_grid_gridprops#toc-sort) by Product name.
+
+    ```js
+      <script>
+        import { ref, defineComponent } from 'vue';
+        import productsData from './appdata/products.json';
+        import { Grid } from '@progress/kendo-vue-grid';
+        import '@progress/kendo-theme-default/dist/all.css';
+
+        export default defineComponent({
+          components: {
+            'grid': Grid,
+          },
+          setup() {
+            const products = productsData;
+            const pageable = ref(true);
+            const sortable = ref(true);
+            const skip = ref(0);
+            const take = ref(10);
+            const sort = ref([
+              { field: "ProductName", dir: "asc" }
+            ]);
+
+            const columns = [
+            { field: 'ProductName', title: 'Product Name' },
+            { field: 'UnitPrice', title: 'Price' },
+            { field: 'UnitsInStock', title: 'Units in Stock' },
+            { field: 'Discontinued' }
+          ];
+
+            return {
+              products, columns, pageable, sortable, skip, take, sort,
+            }
+          }
+        });
+      </script>
+    ```
 
-### VSCode Extension
```

Comment:
As an only addition to the Peko's comments, Im just curious about why we are removing  the "VSCode Extension" and the "Virtual Classroom" sections from the end of the article?Aren't these paths considered as a good getting started point any more?

Reply:
The page path data in Google Analytics shows that these sections aren't popular enough to justify the added article length. As people don't click these links, I recommend moving them to the "Next Steps" section. The reference is still available as we still consider them a good path, it's just that they won't have that prominent space in the article and won't make it feel overwhelmingly long.